opendbc/car/honda/interface.py

```diff
@@ -207,8 +217,8 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
     if ret.openpilotLongitudinalControl and candidate in HONDA_BOSCH:
       ret.safetyConfigs[0].safetyParam |= HondaSafetyFlags.BOSCH_LONG.value
 
-    if candidate in HONDA_BOSCH_RADARLESS:
-      ret.safetyConfigs[0].safetyParam |= HondaSafetyFlags.RADARLESS.value
+    if candidate in (HONDA_BOSCH_RADARLESS | HONDA_CANFD_CAR):
+      ret.safetyConfigs[-1].safetyParam |= HondaSafetyFlags.RADARLESS.value
```

Comment:
We shouldn't mix the safety params. Are you sure this lets you send everything the non-radarless Hondas try to send in openpilot? `0x33DA` is missing in safety.

Do we need to set radarless?

Reply:
Honestly it's been so long since I did the initial port, so I don't remember the specifics on why this was added. I'll remove for the moment.